emqx/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fa40ec7e-e8f6-4c4b-a675-31716b23a9df",
+  "app_id": "emqx",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Emqx integration with openmetrics v2",
+    "title": "EMQX",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "emqx",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "emqx.",
+        "check": ["emqx.cluster.nodes_running"],
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }},
+      "dashboards": {
+        "EMQX Overview": "assets/dashboards/overview.json"
+      }
+  },
+  "author": {
+    "support_email": "zhongwen.deng@emqx.io",
+    "name": "Community",
+    "homepage": "https://github.com/DataDog/integrations-extras",
+    "sales_email": "zhongwen.deng@emqx.io"
```

Comment:
We recommend having an email distribution list here and for support_email rather than an individual